libminifi/include/FlowController.h

```diff
@@ -98,7 +100,7 @@ class FlowController : public core::controller::ControllerServiceProvider, publi
   virtual void load(const std::shared_ptr<core::ProcessGroup> &root = nullptr, bool reload = false);
 
   // Whether the Flow Controller is start running
-  virtual bool isRunning() {
+  bool isRunning() override {
```

Comment:
Minor, but seem logically `const`.

Reply:
I can't fix the const-correctness of overridden virtual functions without changing more classes and potentially breaking API. I wanted to limit my unrelated refactorings to this file.